desktop/src/gui/controller.rs

```diff
@@ -224,10 +229,24 @@ impl GuiController {
     }
 
     pub fn render(&mut self, mut player: Option<MutexGuard<Player>>) {
-        let surface_texture = self
-            .surface
-            .get_current_texture()
-            .expect("Surface became unavailable");
+        let surface_texture = match self.surface.get_current_texture() {
+            Ok(surface_texture) => surface_texture,
+            Err(e @ (SurfaceError::Lost | SurfaceError::Outdated)) => {
+                // Reconfigure the surface if lost or outdated
+                tracing::debug!("Surface became unavailable: {:?}, reconfiguring", e);
```

Comment:
Isn't this more at the level of `tracing::warn`?

Reply:
I was wondering about this, on one hand it _may_ suggest that something's wrong, but on the other hand this is a fairly "standard" error, in a way that it just tells the application that it should recreate the swap chain. My logic was that "if it works, we don't need the logs, but if it's not working we can always debug this by enabling debug level".

So anyway, I'll switch to `warn` for all errors because now that I'm thinking about this, it was a panic before so these errors probably did not happen often :smile: